doc/Caliptra.md

```diff
@@ -8,15 +8,17 @@
 
 <div style="page-break-after: always"></div>
 
-# Foreword
+# Introduction
 
-Caliptra[^1] was originally created as part of the Open Compute Project ([OCP](https://www.opencompute.org/)). The major revisions of the Caliptra specifications are published at OCP. The evolving source code and documentation for Caliptra live in this repository within the [CHIPS Alliance Project](https://chipsalliance.org/), a Series of LF Projects, LLC.
+Caliptra[^1] was originally created as part of the Open Compute Project ([OCP](https://www.opencompute.org/)). The major revisions of the Caliptra specifications are published at OCP. The evolving source code and documentation for Caliptra are in this repository within the [CHIPS Alliance Project](https://chipsalliance.org/), a Series of LF Projects, LLC.
```

Comment:
>a Series of LF Projects, LLC.

->
>a series of Linux Foundation projects. 

Not sure what the LLC is for, CHIPS or Linux Foundation.

Reply:
Out of scope for this commit. Please file an issue.